scripts/get-salt-version.sh

```diff
@@ -30,7 +30,7 @@ compare_version () {
 BASE_NAME=$1
 STACK_VERSION=$2
 
-SALT_VERSION=3001.8
+SALT_VERSION=3004.1
```

Comment:
Isn't 3004.1 something we previously couldn't update to? Or is this an RHEL / Gov related mandatory change? Personally I wouldn't do this together with all of the changes required for Gov, but instead separately under https://jira.cloudera.com/browse/CB-16854 on master and test it with CentOS 7 first.

Reply:
Agree, it came in dunno why... Maybe it was there by a mistake on the base branch I used.